composer.json

```diff
@@ -12,10 +12,10 @@
         "role": "Developer"
     }],
     "require": {
-        "illuminate/contracts": "^6.0",
-        "illuminate/events": "^6.0",
-        "illuminate/filesystem": "^6.0",
-        "illuminate/view": "^6.0",
+        "illuminate/contracts": "^7.0",
```

Comment:
6.0 is still supported until September 3rd, 2022 so I'm not sure it's wise to drop support for it just yet

Reply:
> 6.0 is still supported until September 3rd, 2022 so I'm not sure it's wise to drop support for it just yet

Maybe you should tag a new version for it, Then let the users to choose.  I am using the hyperf framework, it released 2.0 today. For the reason that the framework's lower dependency of symfony components changed from ^4.0 to ^5.0, I need to use the symfony 5.0+（the illuminate needs to use 7.0+）。
Since symfony 4.0 to 5.0, some components's interface is changed, it's not compatible. For all of the above, it's because of the dependency tree, it's need to use 7.0+ of illuminate.

Reply:
Thank your for your advice of .gitignore, i have already removed the ignores that should be global.

Reply:
I think you're suggesting maintaining two versions of this library (one for Laravel 6 and one for Laravel 7), that isn't very attractive to me sorry.
What I'm planning is to support both 6.* and 7.* here, just got a couple of issues to resolve first

Reply:
It's OK，thank you for your reply,  before you support 7's features, I'll use my fork version to compatible.
I am using the [Hyperf framework](https://github.com/hyperf/hyperf), it's a high-performance, highly flexible PHP CLI framework based on Swoole 4.4+. 
Since opensource for one year, it got 2.7K stars. 
It's a coroutine framework and if your are using laravel/lumen,  it will take a very low cost migration from laravel/lumen to hyperf. So I like it very much, I suggest that you can take a look of it.
Hyperf is based on [Swoole](https://github.com/swoole/swoole-src), which has 15.8k stars on github, it makes php possible to do more and has multi interesting features you may like.